AEPCore.xcodeproj/project.pbxproj

```diff
@@ -1682,7 +1687,9 @@
 				BF4A6F2426BB4A1B00612434 /* rules_testDispatchEventNoSource.json */,
 				BF4A6F2526BB4A1B00612434 /* rules_testDispatchEventNoType.json */,
 				BBA5129724F4588E0030DAD1 /* rules_testGroupLogicalOperators.json */,
-				BBA512A424F4A1790030DAD1 /* rules_testMatcherCo.json */,
```

Comment:
build is failing because rules_testMatcherCo.json is missing, should this be added back?